src/liblibc/lib.rs

```diff
@@ -4827,7 +4827,6 @@ pub mod funcs {
                                    mibp: *mut c_int,
                                    sizep: *mut size_t)
                                    -> c_int;
-            pub fn getdtablesize() -> c_int;
```

Comment:
Can you leave these definitions in liblibc? They may not be used by the stdlib, but someone else might be using them!